test/testdrive/load-generator.td

```diff
@@ -29,3 +29,17 @@ users         subsource      <null>
 # Show that AST of subsource contains REFERENCES option
 > SHOW CREATE SOURCE accounts
 materialize.public.accounts "CREATE SUBSOURCE \"materialize\".\"public\".\"accounts\" (\"id\" \"pg_catalog\".\"int8\" NOT NULL, \"org_id\" \"pg_catalog\".\"int8\" NOT NULL, \"balance\" \"pg_catalog\".\"int8\" NOT NULL, UNIQUE (\"id\")) WITH (REFERENCES = true)"
+
+# Check that non-append-only `COUNTER` sources reach the proper size
+
+> CREATE SOURCE counter FROM LOAD GENERATOR COUNTER (MAX CARDINALITY 47, TICK INTERVAL '0.1s') WITH (SIZE '1')
```

Comment:
This will add 4.7s to this test every run. How about reducing the cardinality and tick interval so that this test doesn't block for so long.